### PR TITLE
docs: stable channel is null-safe

### PR DESCRIPTION
**Pull request by russellwheatley**
## Description

Null safety is now available in the `stable` Flutter channel for Web, Android & iOS targets.

~~Null-safe support for [Desktop](https://flutter.dev/desktop) is in `beta` (although there is a "snapshot" in `stable`, I figure it is best to point users to the channel which will be regularly updated).~~

Also documented its availability for Desktop even if it is a "snapshot".

## Related Issues

*Replace this paragraph with a list of issues related to this PR from the [issue database](https://github.com/flutter/flutter/issues). Indicate, which of these issues are resolved or fixed by this PR. Note that you'll have to prefix the issue numbers with flutter/flutter#.*

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`).
This will ensure a smooth and quick review process. Updating the `pubspec.yaml` and changelogs is not required.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [x] All existing and new tests are passing.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] The analyzer (`flutter analyze`) does not report any problems on my PR.
- [x] I read and followed the [Flutter Style Guide].
- [x] I signed the [CLA].
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change.
- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/FirebaseExtended/flutterfire/blob/master/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[CLA]: https://cla.developers.google.com/

### CODE REVIEW

File: docs/analytics/overview.mdx [hunk 1]
```diff
@@ -26,23 +26,23 @@ values={[
 
 :::caution
 This package version is not null safe migrated yet, but has been implemented to be compatible with other null safe FlutterFire packages.
-With that in mind, you will need to use the `beta` channel which supports null safe projects.
+Null safety is now available in Flutter's `stable` channel for Web, Android & iOS targets.
```
Comment (Salakar) [inline docs/analytics/overview.mdx:29]:
```suggestion
```

Don't think we need this now, we say roughly the same thing again a few lines down.

File: docs/database/overview.mdx [hunk 2]
```diff
@@ -27,23 +27,23 @@ values={[
 
 :::caution
 This package version is not null safe migrated yet, but has been implemented to be compatible with other null safe FlutterFire packages.
-With that in mind, you will need to use the `beta` channel which supports null safe projects.
+Null safety is now available in Flutter's `stable` channel for Web, Android & iOS targets.
```
Comment (Salakar) [inline docs/database/overview.mdx:30]:
```suggestion
```

Don't think we need this now, we say roughly the same thing again a few lines down.

File: docs/performance/overview.mdx [hunk 3]
```diff
@@ -25,23 +25,23 @@ values={[
 
 :::caution
 This package version is not null safe migrated yet, but has been implemented to be compatible with other null safe FlutterFire packages.
-With that in mind, you will need to use the `beta` channel which supports null safe projects.
+Null safety is now available in Flutter's `stable` channel for Web, Android & iOS targets.
```
Comment (Salakar) [inline docs/performance/overview.mdx:28]:
```suggestion
```

Don't think we need this now, we say roughly the same thing again a few lines down.

File: docs/remote-config/overview.mdx [hunk 4]
```diff
@@ -29,23 +29,23 @@ values={[
 
 :::caution
 This package version is not null safe migrated yet, but has been implemented to be compatible with other null safe FlutterFire packages.
-With that in mind, you will need to use the `beta` channel which supports null safe projects.
+Null safety is now available in Flutter's `stable` channel for Web, Android & iOS targets.
```
Comment (Salakar) [inline docs/remote-config/overview.mdx:32]:
```suggestion
```

Don't think we need this now, we say roughly the same thing again a few lines down.

File: docs/analytics/overview.mdx [hunk 5]
```diff
@@ -26,23 +26,23 @@ values={[
 
 :::caution
```
Comment (rrousselGit) [inline docs/analytics/overview.mdx:27]: Now that analytics is migrated, we can probably remove the entire warning

File: docs/analytics/overview.mdx [hunk 6]
```diff
@@ -26,23 +26,23 @@ values={[
...
 This package version is not null safe migrated yet, but has been implemented to be compatible with other null safe FlutterFire packages.
-With that in mind, you will need to use the `beta` channel which supports null safe projects.
+Null safety is now available in Flutter's `stable` channel for Web, Android & iOS targets.
 :::
```
Comment (Salakar) [inline docs/analytics/overview.mdx:30]: Lets remove this caution now - analytics has just been migrated

**Review by rrousselGit — APPROVED** [review]
LGTM

**Review by Salakar — APPROVED** [review]

**Pull request merged**